src/ext_event.rs

```diff
@@ -158,6 +159,7 @@ pub fn update_signal_from_channel<T: Send + 'static>(
     });
 }
 
+#[cfg(feature = "crossbeam")]
 pub fn create_signal_from_channel<T: Send + 'static>(
     rx: crossbeam_channel::Receiver<T>,
```

Comment:
Wouldn't it be better to put

```suggestion
    #[cfg(feature = "crossbeam")]
    rx: crossbeam_channel::Receiver<T>,
    #[cfg(feature = "crossbeam")]
    rx: std::sync::mpsc::Receiver<T>,
```

Reply:
Fixing now.